src/algebra/order/to_interval_mod.lean

```diff
@@ -426,6 +427,46 @@ to_Ico_mod_add_right a hb
 lemma to_Ioc_mod_periodic (a : α) {b : α} (hb : 0 < b) : function.periodic (to_Ioc_mod a hb) b :=
 to_Ioc_mod_add_right a hb
 
+/-- `to_Ico_mod` as an equiv from the quotient. -/
+@[simps symm_apply]
+def quotient_add_group.equiv_Ico_mod (a : α) {b : α} (hb : 0 < b) :
+  (α ⧸ add_subgroup.zmultiples b) ≃ set.Ico a (a + b) :=
```

Comment:
Would it make sense to generalise `add_circle` (by removing the unused arguments) to avoid duplicating this definition?

Reply:
It's not really duplicated, the `add_circle` version is less general.

Reply:
Oh I see. But somehow it's less general because of the RHS while I was talking about the LHS. My point is that I want it to read
```suggestion
  add_circle b ≃ set.Ico a (a + b) :=
```

Reply:
I agree in principle, but was deliberately avoiding doing that in this PR because I didn't want to scope creep to changing the API of `add_circle`. I can only assume that those unused arguments are deliberate and that removing them would therefore be contentious.

Reply:
I agree in principle 😛 

maintainer merge